### PR TITLE
Print threat weights directly as i8.

### PR DESCRIPTION
**Pull request by sscg13**
Companion PR to https://github.com/official-stockfish/Stockfish/pull/6479

### CODE REVIEW

File: model/utils/serialize.py [hunk 1]
```diff
@@ -310,7 +316,12 @@ def read_feature_transformer(
 
         bias = self.tensor(np.int16, [layer.bias.shape[0] - num_psqt_buckets])
         # weights stored as [num_features][outputs]
-        weight = self.tensor(np.int16, [shape[0], shape[1] - num_psqt_buckets])
+        if self.feature_set.name.startswith("Full_Threats"):
+            threat_weight = self.tensor(np.int8, [79856, shape[1] - num_psqt_buckets])
+            psq_weight = self.tensor(np.int16, [shape[0] - 79856, shape[1] - num_psqt_buckets])
+            weight = torch.cat(threat_weight, psq_weight, dim=0)
```
Comment (Disservin) [inline model/utils/serialize.py]: i think you need an array for torch.cat?
`torch.cat([threat_weight, psq_weight], dim=0)`

Comment (sscg13) [reply model/utils/serialize.py]: ah oops
